svelte-kit/src/lib/components/ModalField.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+    export let isTitle = false;
+    export let text: string;
+
+    let active = false;
+    let input: HTMLElement;
+
+    const toggleField = () => {
+        active = true;
+        input.focus();
+    };
+</script>
+
+<div class:active class="input-group | flex items-center gap-5 justify-between">
+    {#if isTitle}
+        <h3 class="input-group__text">{text}</h3>
```

Comment:
```suggestion
        <h2 class="input-group__text">{text}</h2>
```

I think `h2` is more appropriate for a modal title as it is technically at the top of the DOM tree (but cannot be a `h1` for obvious reasons).